src/main/java/com/seailz/discordjar/model/message/Message.java

```diff
@@ -420,5 +438,54 @@ public String getFormattedText() {
             formatted = formatted.replaceAll("<@&" + role.id() + ">", "@" + role.name());
         return formatted;
     }
+
+    public record StickerItem(
+            String id,
+            String name,
+            StickerFormat format
+    ) implements Compilerable {
+
+        @Override
+        public JSONObject compile() {
+            return new JSONObject()
+                    .put("id", id)
+                    .put("name", name)
+                    .put("format_type", format.getCode());
+        }
+
+        public static StickerItem decompile(JSONObject obj) {
+            return new StickerItem(
+                    obj.getString("id"),
+                    obj.getString("name"),
+                    StickerFormat.getStickerFormatByCode(obj.getInt("format_type"))
+            );
+        }
+    }
+
+    public record RoleSubscriptionData(
+            String roleSubscriptionListingId,
+            String tierName,
+            int totalMonthsSubbed,
+            boolean isRenewal
+    ) implements Compilerable {
+
+            @Override
+            public JSONObject compile() {
+                return new JSONObject()
+                        .put("role_subscription_listing_id", roleSubscriptionListingId)
+                        .put("tier_name", tierName)
+                        .put("total_months_subbed", totalMonthsSubbed)
```

Comment:
`total_months_subbed` is `subscribed` in the documentation, did you do that on purpose for some reason? if you did then don't mind what I suggested

Reply:
In the documentation I'm looking at, it's `total_months_subscribed`, so you've actually caught a bug - just a different one lol.

Will fix in a bit